taxonomies/food/ingredients.txt

```diff
@@ -52397,20 +52402,42 @@ wikidata:en: Q35075688
 < en:green asparagus
 es: Espárragos verdes sin pelar
 
+< en:asparagus
+en: violet asparagus
+fr: asperge violette
+
 < en:asparagus
 en: white asparagus
 bg: бяла аспержа, бели аспержи
-de: weisse Spargeln
+de: weisse Spargeln, weißer Spargel
 es: Espárrago blanco
 fi: valkoinen parsa
-fr: asperges blanches, asperge blanche
+fr: asperges blanches, asperge blanche, asperges blanches grosses, asperges blanches des sables des Landes IGP
 hr: bijele šparoge
 it: asparagi bianchi
 nl: witte asperges
 pl: białe szparagi, biały szparag, białych szparagów
 pt: Espargos brancos
 # fr:asperges blanches has 46 products in 4 languages @2018-11-11
 
+< en:white asparagus
+fr: asperges blanches coupées
```

Comment:
we should not need this one, "coupées" is in processing

Reply:
agree, reverted